### DISCUSSION TITLE
http://localhost:1313/posts/notes/dev/auth%E5%BE%AE%E6%9C%8D%E5%8A%A1%E5%BC%80%E5%8F%91%E8%AE%B0%E5%BD%95/

### DISCUSSION THREAD

**Discussion by giscus** [yunyit/yunyit.github.io#143] [Comments]
# http://localhost:1313/posts/notes/dev/auth%E5%BE%AE%E6%9C%8D%E5%8A%A1%E5%BC%80%E5%8F%91%E8%AE%B0%E5%BD%95/

引入刷新令牌机制

http://localhost:1313/posts/notes/dev/auth%E5%BE%AE%E6%9C%8D%E5%8A%A1%E5%BC%80%E5%8F%91%E8%AE%B0%E5%BD%95/

**Comment by sirius2alpha** [comment]
感觉还可以再拓展一点
